php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2070.html

```diff
@@ -1,7 +1,7 @@
 <p>The MD5 algorithm and its successor, SHA-1, are no longer considered secure, because it is too easy to create hash collisions with them. That is,
 it takes too little computational effort to come up with a different input that produces the same MD5 or SHA-1 hash, and using the new, same-hash
 value gives an attacker the same access as if he had the originally-hashed value. This applies as well to the other Message-Digest algorithms: MD2,
-MD4, MD6, HMAC-MD5, DSA (which uses SHA-1), RIPEMD160, HMACRIPEMD160.</p>
+MD4, MD6, HAVAL-128, HMAC-MD5, DSA (which uses SHA-1), RIPEMD, RIPEMD-128, RIPEMD-160, HMACRIPEMD160.</p>
```

Comment:
the rule must be updated with the new algorithms

Reply:
in fact rule only raises for sha1 and md5. So i would consider this list of algorithms as documentation, as looks like there is no standard way to use them in php